tests/rptest/services/action_injector.py

```diff
@@ -42,13 +52,20 @@ class DisruptiveAction:
     The action can be reversible, it also stores the set of affected nodes and the last node
     the action was applied on.
     """
+
+    is_reversible = False
+
     def __init__(self, redpanda: RedpandaService, config: ActionConfig,
                  admin: Admin):
         self.admin = admin
         self.config = config
         self.redpanda = redpanda
+        self.nodes_for_action = queue.PriorityQueue(
```

Comment:
nit: PriorityQueue seems like an overkill because we don't need any synchronization here. Although if its interface is more convenient than the raw `heapq` module, then it is okay I guess.

Reply:
Yes, it is mostly for the PriorityQueue API, we always read from it in non-blocking mode although now looking at the code we have max size and we put in blocking mode, maybe it should be fixed. 

If you feel `heapq` should be used let me know, I can try a refactor.